### ISSUE TITLE
[Request] Specific Whitelisting

### ISSUE THREAD

**Issue by zero77** [kkapsner/CanvasBlocker#530]
## Description

Could you please give the option to temporary whitelist only what you want and not everything, per domain or page.
Currently unless i have missed something, if you temporary whitelist something everything gets whitelisted for that domain or page.

## Steps to Reproduce (for bugs)

N/A

## Your Environment

Any

**Comment by kkapsner** [comment]
Yes. Currently everything get's whitelisted if you do it temporarily.
